packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -811,6 +816,7 @@ class _PopupMenuRoute<T> extends PopupRoute<T> {
 
   final RelativeRect position;
   final List<PopupMenuEntry<T>> items;
+  final List<GlobalKey>? itemKeys;
```

Comment:
Are there any other users of `_PopupMenuRoute`? If not, this doesn't need to be nullable.

Reply:
You are right. Thanks for catching this! Updating.